compiler/AST/flags.cpp

```diff
@@ -95,6 +96,31 @@ viewFlags(BaseAST* ast) {
         printf("\n");
       }
     }
+    if (viewFlagsExtras) {
```

Comment:
I'm not sure how much I like having these extras tied to viewFlags, but I think I can live with it.